ansible/provision-tiered-storage-cluster.yml

```diff
@@ -33,3 +33,24 @@
       ansible.builtin.include_role:
         name: redpanda.cluster.redpanda_broker
       when: not skip_node | default(false) | bool
+
+- name: Provision redpanda console
+  hosts: client
+  vars:
+    advertise_public_ips: true
+    redpanda_version: latest
+    install_console: true
+    enable_tls: true
+    create_demo_certs: true
+    handle_cert_install: true
```

Comment:
[question]

Will this override what's passed via flags? I know these are not intended for production use. Just wondering how does the var override works

Reply:
Command line -> hosts file -> playbook iirc. But I can guarantee command line is supreme.